common/config/azure-pipelines/templates/record-published-versions.yaml

```diff
@@ -16,6 +16,7 @@ steps:
   - script: 'node $(Agent.BuildDirectory)/rushstack/repo-scripts/repo-toolbox/lib/start.js record-versions --out-file $(Build.ArtifactStagingDirectory)/published-versions/published-versions.json'
     displayName: 'Record Published Versions'
     workingDirectory: '$(Agent.BuildDirectory)/rushstack-legacy'
-  - publish: $(Build.ArtifactStagingDirectory)/published-versions
-    artifact: published-versions
-    displayName: 'Publish Artifact: published-versions'
+  # Published by the 1ES template

```

Comment:
Intentional?

Reply:
Yes. This is the same as the rushstack one.